missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf

```diff
@@ -0,0 +1,525 @@
+// Info: Finnish Army 1939 Loadouts
+// - For multi line definitions use \
+// - All loadouts listed are customised to correct equipment, so have fun with them
+// - ALWAYS ADD SHORT-RANGE RADIO FIRST
+// - On any random equipment definition remember to use Olsen_FW_FNC_AddItemRandom
+
+/* Loadouts
+
+    //Platoon HQ
+[this, F39_PC] call Olsen_FW_FNC_GearScript;          Platoon Commander
+[this, F39_PSgt] call Olsen_FW_FNC_GearScript;        Platoon Sergeant
+[this, F39_Mess] call Olsen_FW_FNC_GearScript;        Messenger
+[this, F39_MedP] call Olsen_FW_FNC_GearScript;        Medic
+[this, F39_RTO] call Olsen_FW_FNC_GearScript;         Radio Operator
+
+    //Squad
+[this, F39_SL] call Olsen_FW_FNC_GearScript;          Squad Leader
+[this, F39_TL] call Olsen_FW_FNC_GearScript;          Team Leader
+[this, F39_SMG] call Olsen_FW_FNC_GearScript;         Submachine Gunner
+[this, F39_Gren] call Olsen_FW_FNC_GearScript;        Grenadier
+[this, F39_MedS] call Olsen_FW_FNC_GearScript;        Assistant Medic
+
```

Comment:
Is there a whiteline here or is that Github fucking up the rendering?

Reply:
I copied that top part from the Russian loadouts, they seem to have the same. I got it of the empty line in these loadouts